src/foundry/foundry.js/clientDocuments/fogExploration.d.ts

```diff
@@ -1,12 +1,54 @@
-//TODO: This is just a stub
+import { ConfiguredDocumentClass } from '../../../types/helperTypes';
+import { RequestOptions } from '../../common/abstract/backend.mjs';
+
 declare global {
   /**
    * The client-side FogExploration document which extends the common BaseFogExploration model.
    * Each FogExploration document contains FogExplorationData which defines its data schema.
    *
-   * @see {@link data.FogExplorationData}              The FogExploration data schema
+   * @see {@link data.FogExplorationData} The FogExploration data schema
    */
-  class FogExploration extends ClientDocumentMixin(foundry.documents.BaseFogExploration) {}
+  class FogExploration extends ClientDocumentMixin(foundry.documents.BaseFogExploration) {
+    /**
+     * Explore fog of war for a new point source position.
+     * @param source - The candidate source of exploration
+     * @param force  - Force the position to be re-explored
+     *                 (default: `false`)
+     * @returns Is the source position newly explored?
+     */
+    explore(source: PointSource, force?: boolean): boolean;
+
+    /**
+     * Obtain the fog of war exploration progress for a specific Scene and User.
+     * @param options - (default: `{}`)
+     */
+    static get(
+      {
+        scene,
+        user
+      }?: {
+        scene?: InstanceType<ConfiguredDocumentClass<typeof Scene>>;
+        user?: InstanceType<ConfiguredDocumentClass<typeof User>>;
+      },
+      options?: RequestOptions
+    ): Promise<StoredDocument<InstanceType<ConfiguredDocumentClass<typeof FogExploration>>> | null>;
+
+    /**
+     * Transform the explored base64 data into a PIXI.Texture object
+     */
+    getTexture(): PIXI.Texture | null;
+
+    /**
+     * Open Socket listeners which transact JournalEntry data
+     */
+    protected static _activateSocketListeners(socket: io.Socket): void;
```

Comment:
I think it shouldn't actually be protected, similar to other activateSocketListeners functions. But apparently they are part of the private API, so it should be marked `@internal`.

Reply:
You're right, it is called from the game. :+1: